CHANGELOG.md

```diff
@@ -7,3 +7,13 @@ and this project adheres to [Semantic
 Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+
+## [0.0.1] - 2022-03-18
+
+### Added
+
+- Add openfun's rules to commit format
+
+### Changed
+
+- Fix several bugs in CI to make a clean starting repo
```

Comment:
this should not be part of the changelog which decribes changes in the application, not in the project tools.
The check-changelog task is not required and we can merge even if it fails. It's just here as a reminder.